src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -328,6 +328,8 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*CpuReservationTest.*Mutiple_CompiledModel_Reservation.*)",
         // Issue: 163351
         R"(.*CoreThreadingTestsWithIter.*nightly_AsyncInfer_ShareInput.*)",
+        // Interpolate used in the test not support all test precisions
+        R"(.*importExportModelWithTypeRelaxedExtension.*elementType=(i16|u16|u32|u64).*)",
```

Comment:
I hardly believe `Interpolate` will ever support `(i16|u16|u32|u64)`. It's more meaningful to remove these precisions from the test case at all.

Reply:
The test fixture is used for other tests also, so remove precision may be not good option.
If this line must be removed I need to add dedicated instance of fixture with different precision list.

Reply:
Test updated and this regex removed.